### DISCUSSION TITLE
Interoperability with jsonresume.org

### DISCUSSION THREAD

**Discussion by isanchezd** [getmanfred/mac#16] [Ideas]
Hi!

First of all, congratulations for the initiative and I wish you good luck :)

Second thing, today i heard about a project called https://jsonresume.org/  that apparently  their idea is create an oriented JSON format to resumes.

They have a little ecosystem of tools with their format.

I suggest it might be interesting consider adopt an interoperability between MAC and jsonresume because 2 standard formats on the market that be painfull for the people that are interested in the idea of ​​consuming a single json format and to take advantage of the ecosystem of tools.

Thank you!

**Comment by jrub** [comment]
Thanks for sharing @isanchezd, in fact that was my first thought when I heard about Manfred's initiative :-) jsonresume has been around for almost a decade. Offering compatibility or translation with jsonresume would provide MAC users with their HTML and PDF themes for exporting CVs in case we'd want to do that, or even to other tools like https://github.com/joshuatz/linkedin-to-jsonresume
